app/scripts/modules/amazon/src/function/configure/FunctionBasicInformation.tsx

```diff
@@ -0,0 +1,271 @@
+    const { accounts, regions } = this.state;
+    const className = classNames({
+      well: true,
+      'alert-danger': errors.functionName,
+      'alert-info': !errors.functionName,
+    });
+    return (
+      <div className="container-fluid form-horizontal ">
+        <div className="sp-margin-m-bottom">
+          {isNew && (
+            <div className={className}>
+              <strong>Your load balancer will be named: </strong>
+              <HelpField id="aws.function.name" />
+              <span>{this.props.app.name.concat('-').concat(values.functionName)}</span>
+              <Field
+                type="text"
+                style={{ display: 'none' }}
+                className="form-control input-sm no-spel"
+                name="functionName"
+              />
+              {errors.functionName && <ValidationMessage type="error" message={errors.functionName} />}
+            </div>
+          )}
+        </div>
+        <div className="sp-margin-m-bottom">
+          <FormikFormField
+            fastField={false}
+            name="credentials"
+            label="Account"
+            input={props => (
+              <ReactSelectInput
+                {...props}
+                inputClassName="cloudfoundry-react-select"
+                value={values.credentials}
+                stringOptions={accounts.map((acc: IAccount) => acc.name)}
+                clearable={true}
+              />
+            )}
+          />
+        </div>
+        <div className="sp-margin-m-bottom">
+          <RegionSelectField
+            labelColumns={3}
+            component={values}
+            field="region"
+            account={values.credentials}
+            onChange={this.handleRegionChange}
+            regions={regions}
+          />
+        </div>
+        <div className="sp-margin-m-bottom">
+          <FormikFormField
+            name="functionName"
+            label="Function Name"
+            help={<HelpField id="aws.function.name" />}
+            input={props => <TextInput {...props} />}
+          />
+        </div>
+        <div className="sp-margin-m-bottom">
+          <FormikFormField
+            name="runtime"
+            label="Runtime"
+            help={<HelpField id="aws.function.runtime" />}
+            input={props => (
+              <ReactSelectInput
+                {...props}
+                inputClassName="cloudfoundry-react-select"
+                stringOptions={availableRuntimes}
+                clearable={true}
+              />
+            )}
+          />
+        </div>
+        <div className="sp-margin-m-bottom">
+          <FormikFormField
+            name="s3bucket"
+            label="S3 Bucket"
+            help={<HelpField id="aws.function.s3bucket" />}
+            input={props => <TextInput {...props} placeholder="S3 bucket name" />}
+          />
+        </div>
+        {errors.s3bucket && <ValidationMessage type="error" message={errors.s3bucket} />}
```

Comment:
`FormikFormField` should be rendering any error already, so probably delete this line.

Reply:
It does not display the error message. 
However, to be consistent, I am deleting it.